moveos/moveos/src/moveos.rs

```diff
@@ -175,6 +193,144 @@ impl MoveOS {
         }
     }
 
+    fn deserialize_modules(&self, module_bytes: &[Vec<u8>]) -> Result<Vec<CompiledModule>> {
+        let modules = module_bytes
+            .iter()
+            .map(|b| CompiledModule::deserialize(b).map_err(|e| e.finish(Location::Undefined)))
+            .collect::<VMResult<Vec<CompiledModule>>>()
+            .map_err(|e| anyhow!("Failed to deserialize modules: {:?}", e))?;
+
+        assert!(
+            !modules.is_empty(),
+            "input checker ensures package is not empty"
+        );
+
+        Ok(modules)
+    }
+
+    /// The initializer function must have the following properties in order to be executed at publication:
+    /// - Name init
+    /// - Single parameter of &mut TxContext type
```

Comment:
Is support `signer`? if not, can not call `global_move_to`

Reply:
`init` function doesn't support `signer`, so can not `global_move_to `